git-extra/astextplain

```diff
@@ -8,6 +8,9 @@ fi
 
 # XXX output encoding (UTF-8) hardcoded
 case "$1" in
+	*.ods | *.ODS | *odf |*.ODF | *odt | *.ODT)
+		odf2txt "$1" || cat "$1"
```

Comment:
You can, of course, use the PKGBUILD to which you linked as example, but please note a couple of crucial, required changes, as we are talking about a MINGW package:

- [`arch` must be `any`](https://github.com/git-for-windows/MINGW-packages/commit/898f5ed4bd2c6f5addd60aabb977051d97d0350f#diff-312c6a788fc5432403915ac0aa4090b6R10), as you can install 32-bit and 64-bit MINGW packages side-by-side,
- MINGW executables [install into `$MINGW_PREFIX/bin`](https://github.com/git-for-windows/MINGW-packages/commit/898f5ed4bd2c6f5addd60aabb977051d97d0350f#diff-312c6a788fc5432403915ac0aa4090b6R68), not `/usr/bin/`, and
- It will be `odt2txt.exe`, i.e. [you will have to add the `.exe` extension](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/odt2txt#n25)

Please also note that I'd much rather use a tag than a commit, if at all possible. If not, I'd rather use `master` [with a custom-generated `pkgver` reflecting the number of commits and the short commit name](https://github.com/git-for-windows/MINGW-packages/commit/898f5ed4bd2c6f5addd60aabb977051d97d0350f#diff-312c6a788fc5432403915ac0aa4090b6R19).